fhirest-core/src/main/java/ee/fhir/fhirest/core/service/conformance/HapiContextHolder.java

```diff
@@ -121,7 +128,13 @@ protected IValidationSupport getValidationSupport() {
         .filter(c -> SUPPORTED_CS_CONTENT_MODES.contains(c.getContent()))
         .collect(Collectors.toMap(d -> d.getUrl(), d -> d, (a, b) -> a));
 
+    CacheConfiguration cacheConfiguration = defaultValues();
```

Comment:
in this specific case, `CacheConfiguration.defaultValues()` would be more readable, without static import